src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -153,19 +156,6 @@ public Viewport3DX View
         public PhongMaterial SelectedMaterial { get; set; }
 
         public Transform3D Model1Transform { get; private set; }
-        
-        public RenderTechnique RenderTechnique
```

Comment:
This property is removed as it is no longer required to bind a `RenderTechnique` to the helix view. Each of the types we render, points, curves, and meshes, sets their own `RenderTechnique`. This render technique for the scene is not required.